commands/issue.go

```diff
@@ -603,6 +618,58 @@ text is the title and the rest is the description.`, project))
 	messageBuilder.Cleanup()
 }
 
+func addComment(cmd *Command, args *Args) {
+	issueNumber := ""
+	if args.ParamsSize() > 0 {
+		issueNumber = args.GetParam(0)
+	}
+	if issueNumber == "" {
+		utils.Check(cmd.UsageError(""))
+	}
+
+	localRepo, err := github.LocalRepo()
+	utils.Check(err)
+
+	project, err := localRepo.MainProject()
+	utils.Check(err)
+
+	gh := github.NewClient(project.Host)
+
+	commentBuilder := &github.CommentBuilder{
+		Filename: "ISSUE_EDITMSG",
+	}
+
+	flagIssueEdit := args.Flag.Bool("--edit")
+	flagIssueMessage := args.Flag.AllValues("--message")
+	if len(flagIssueMessage) > 0 {
+		commentBuilder.Message = strings.Join(flagIssueMessage, "\n\n")
+		commentBuilder.Edit = flagIssueEdit
+	} else if args.Flag.HasReceived("--file") {
+		commentBuilder.Message, err = msgFromFile(args.Flag.Value("--file"))
+		utils.Check(err)
+		commentBuilder.Edit = flagIssueEdit
+	} else {
+		commentBuilder.Edit = true
+	}
+
+	message, err := commentBuilder.Extract()
+	utils.Check(err)
+
+	args.NoForward()
+	if args.Noop {
+		ui.Printf("Would add a comment, to the issue %s: %s", issueNumber, message)
```

Comment:
How about simplifying this sentence and having a newline between first line and comment text?
```suggestion
		ui.Printf("Would comment on issue %s:\n%s", issueNumber, message)
```